docs/examples/DataSet/Performing-measurements-using-qcodes-parameters-and-dataset.ipynb

```diff
@@ -172,7 +172,7 @@
     }
    ],
    "source": [
-    "meas = Measurement(exp=exp)\n",
+    "meas = Measurement(exp=exp, name='xyz_measurement')\n",
```

Comment:
more context-friendly names :) and perhaps more occurences of the use of `name=` in this notebook would also help promoting the feature?
```suggestion
    "meas = Measurement(exp=exp, name='exponential_decay')\n",
```

Reply:
ok, yes :)